shared/macros-oval.jinja

```diff
@@ -0,0 +1,236 @@
+{{#
+  High level macro which checks if a particular combination of parameter and value in a configuration file is set.
+  This function can take nine parameters:
+    - path (String): Path to the configuration file to be checked.
+    - prefix_regex (String): Regular expression to be used in the beginning of the OVAL text file content check.
+    - parameter (String): The parameter to be checked in the configuration file.
+    - value (String): The value to be checked. This can also be a regular expression (e.g: value1|value2 can match both values).
+    - separator_regex (String): Regular expression to be used as the separator of parameter and value in a configuration file. If spaces are allowed, this should be included in the regular expression.
+    - default_pass (boolean): Implies that the value for the parameter is the default value. If set, the check will pass even if the parameter is not set in the configuration file.
```

Comment:
I changed it to `parameter_missing_pass` because it is literally what it will do, it can be that default values doesn't exist for a particular application, and simply by not setting the configuration is enough. But also, this serve if the application is configured correctly only using default values which means not overriding them in the configuration file. IMO `parameter_missing_pass` fits better.